apps/access/tests.py

```diff
@@ -89,10 +89,14 @@ def setUp(self):
         self.addon = Addon.objects.get(id=3615)
         self.au = AddonUser.objects.get(addon=self.addon, user=self.user)
         assert self.au.role == amo.AUTHOR_ROLE_OWNER
-        self.request = mock.Mock()
-        self.request.groups = ()
-        self.request.amo_user = self.user
-        self.request.user.is_authenticated.return_value = True
+        self.request = self.fake_request_with_user(self.user)
+
+    def fake_request_with_user(self, user):
+        request = mock.Mock()
+        request.groups = ()
+        request.user = user
+        request.user.is_authenticated = mock.Mock(return_value=True)
```

Comment:
It feels like there should be an easier way but I can't find one. Maybe making a context manager that does this and uses `mock.patch` but that seems a bit overkill.